lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -129,10 +130,16 @@ export class AuthorizationCodeClient extends BaseClient {
      * @param request
      */
     private async executeTokenRequest(authority: Authority, request: AuthorizationCodeRequest): Promise<NetworkResponse<ServerAuthorizationTokenResponse>> {
+        const thumbprint = new RequestThumbprint(
+            this.config.authOptions.clientId,
+            request.authority,
```

Comment:
should use the authority passed into the function instead of the request.authority here.

Reply:
`authority.canonicalAuthority`

Reply:
Done